MediaManager/Plugin.MediaManager.Abstractions/Implementations/MediaManagerBase.cs

```diff
@@ -255,7 +291,8 @@ private void OnPlayingChanged(object sender, PlayingChangedEventArgs e)
         {
             if (sender != CurrentPlaybackManager) return;
             MediaFinished?.Invoke(sender, e);
-            await PlayNext();
+            if (MediaQueue.Repeat == RepeatType.RepeatOne) await Seek(TimeSpan.Zero);
```

Comment:
But this then is a state of the `MediaPlayer`, and not of the queue. We should be clear about this and also reflect this in how we write the interfaces.

If we implement it like this, the `RepeatType` for belongs into a ValueConverter and or the implementation of `IMediaPlayer` but not in an implementation of `IQueue`.

We have the same conflict when it comes to `PlayPrevious()` which either plays the previous item of the queue or jumps to the start of the current item. There are cases, where you (no matter how long the current track already has been played) want to play the previous track.